bin/ColorScript/Sources/ColorScriptCore/Sources/ColorScript.swift

```diff
@@ -0,0 +1,34 @@
+// swiftlint:disable force_try
+import Foundation
+
+public enum ColorScriptError: Error {
+  case missingArguments
+}
+
+public final class ColorScript {
+  private let arguments: [String]
+
+  public let inPath = "Resources/Colors.json"
+  public let outPath = "Resources/Colors.swift"
```

Comment:
Hm we should probably keep the output path the same as where the `Colors.swift` file currently lives. I believe it's at `/Library/Styles/Colors.swift`.

Reply:
It makes sense. I thought of `Resources/` as a replacement for the `Design/` folder (used by the old script). But I just noticed that we have 2 identical `Colors.swift`. One at `Design/Colors.swift` and the other at `Styles/Colors.swift`. I'm wondering what's the motivation behind it.